Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Coordinate/GetByStreetcodeId/GetCoordinatesByStreetcodeIdHandler.cs

```diff
@@ -30,10 +33,14 @@ public async Task<Result<IEnumerable<StreetcodeCoordinateDTO>>> Handle(GetCoordi
 
         if (coordinates is null)
         {
-            return Result.Fail(new Error($"Cannot find a coordinates by a streetcode id: {request.StreetcodeId}"));
+            string errorMsg = $"Cannot find a coordinates by a streetcode id: {request.StreetcodeId}";
+            _logger?.LogError("GetCoordinatesByStreetcodeIdQuery handled with an error");
+            _logger?.LogError(errorMsg);
+            return Result.Fail(new Error(errorMsg));
         }
 
         var coordinatesDto = _mapper.Map<IEnumerable<StreetcodeCoordinateDTO>>(coordinates);
+        _logger?.LogInformation($"GetCoordinatesByStreetcodeIdQuery handled successfully");
```

Comment:
I find logging that information useless. I think we better discuss what kind of information we will log. But certainly logging successful method calls is redundant